phases/ephemeral/witx/wasi_ephemeral_profile.witx

```diff
@@ -0,0 +1,27 @@
+(use "wasi_ephemeral_args.witx")
+(use "wasi_ephemeral_clock.witx")
+(use "wasi_ephemeral_environ.witx")
+(use "wasi_ephemeral_fd.witx")
+(use "wasi_ephemeral_path.witx")
+(use "wasi_ephemeral_poll.witx")
+(use "wasi_ephemeral_proc.witx")
+(use "wasi_ephemeral_random.witx")
+(use "wasi_ephemeral_sched.witx")
+(use "wasi_ephemeral_sock.witx")
+
+;;; The WASI standard specifies executables as having all of the
+;;; WASI modules available for import, and exporting a start function.
+(profile $wasi_ephemeral_executable
+  ;;; Executables must export a start function:
+  (@interface func (export "_start"))
```

Comment:
In the rest of witx (and wit), "export" means something that the API exports, and "import" means something that the API expects you to export so that the API can import it from you. Here, "_start" is something the API expects you to export, so it seems like "import" would be a better way to describe this.

Reply:
I agree this is inconsistent. Part of my motivation was to re-use the existing InterfaceFunc parser and AST. I have been thinking as profile as a predicate a module can satisfy, and from that perspective I think the way export and import are used make sense, but I can be convinced otherwise.

Reply:
Independently of import vs. export, could you move "_start" into its own regular witx module and `(expose ...)` it here? We'll likely need that as soon as we go to define another command-like profile. And I think it would also happen to sidestep this whole question of import vs. export in a profile, because then "_start" would just be an import in a witx file similar to "memory", and the profile would just contain `expose`s.

Reply:
How about we use `(require (@interface func (export "name")...))` - that will correspond to `(expose (module...))`

Reply:
That also leaves us open to `(optional (@interface func ...))` which I could see being useful in a reactor context like proxy-wasms

Reply:
Implemented in https://github.com/WebAssembly/WASI/pull/267/commits/77229818a40dae9be3335731d045fcd59e82ab80